draft-ietf-rtcweb-jsep.xml

```diff
@@ -1977,13 +1977,205 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+              <t>Handle RTP attributes:
+                <list style="symbols">
+                  <t>Read fmtp attributes</t>
+                  <t>Read ssrc-group attributes</t>
+                  <t>Read ssrc attributes</t>
+                  <t>Read ptime, maxptime</t>
+                  <t>Read header extensions</t>
+                  <t>Read rtcp-fb</t>
+                  <t>Read direction attribs (recvonly, etc)</t>
+                  <t>Read rtcp-mux</t>
+                </list>
+              </t>
+              <t>Handle SCTP attributes:
+                <list style="symbols">
+                  <t>Read sctpmap/port attributes</t>
+                </list>
+              </t>
+            </list>
+          </t>
+        </section>
+        <section title="Semantics Verification">
+          <t>Assuming parsing completes successfully, the parsed description is
+            then evaluated to ensure internal consistency as well as proper support
+            for mandatory features. Specifically, the following checks are
+            performed:
+              <list style="symbols">
+                <t>For each m= section, valid values for each of the
+                mandatory-to-use features enumerated in <xref target="sec.usage-requirements"/> MUST be present. These values MAY either be present at the
+                media level, or inherited from the session level.
+                  <list style="symbols">
+                    <t>ICE ufrag and password values</t>
+                    <t>DTLS fingerprint and setup values</t>
+                  </list>
+                </t>
+                <t>If a BUNDLE group is present, the referenced m= sections MUST
+                exist. If multiple BUNDLE groups exist, each referenced m=
+                section MUST only exist in a single BUNDLE group. [TODO: should this apply to all groups?]</t>
+              </list>
+          </t>
+          <t>If this session description is of type "pranswer" or "answer", the
+          following additional checks are applied:
+            <list style="symbols">
+              <t>The number of m= sections MUST exactly match the number of m=
+              sections in the associated offer.</t>
+              <t>For each m= section, the protocol value MUST exactly match the
+              protocol value in the corresponding m= section in the associated
+              offer.</t>
+            </list>
+          </t>
+        </section>
+      </section>
 
-      <section title="Applying a Remote Description" anchor="sec.applying-a-remote-descrip"></section>
+      <section title="Applying a Local Description" anchor="sec.applying-a-local-desc">
+       <t>The following steps are performed at the media engine level to apply a local
+       description. For each m= section, do the following: [TODO - expand]
+         For all m= sections:
+         <list style="symbols">
+          <t>If ufrag/pwd changed, trigger ICE restart</t>
```

Comment:
Agreed. Will speak to that when I flesh this out. Added TODO for now; fortunately we have Section 6 which I can just reference specifically.